jupyterlab_comments/jupyterlab_comments/handlers_test.py

```diff
@@ -0,0 +1,233 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import os
+import tempfile
+import handlers
+import subprocess
+import unittest
+import json
+import tornado.ioloop
+import tornado.web
+import tornado.testing
+
+remote_dir = tempfile.TemporaryDirectory()
+not_git_dir = tempfile.TemporaryDirectory()
+dir1 = tempfile.TemporaryDirectory()
+dir2 = tempfile.TemporaryDirectory()
+dir1_path = dir1.name
+dir2_path = dir2.name
+
+
+home = os.path.expanduser("~")
+FNULL = open(os.devnull, 'w')
+
+def create_files(num_files, dir_name):
+    for i in range(1, num_files + 1):
+        with open(os.path.join(dir_name, "test" + str(i) + ".py"), 'a') as temp_file:
```

Comment:
Calling this `temp_file` confused me at first as I was thinking that it was a file that only existed in the scope of the `with` statement (which is not the case).

I would suggest renaming this to something like `test_file`.